### PR TITLE
CKAN v2.9 and Python3 support

### PR DESCRIPTION
**Pull request by smotornyuk**

### CODE REVIEW

**Comment by smotornyuk** [comment]
Tests are green - ready for review

**Comment by jqnatividad** [comment]
Hi @smotornyuk , you may want to look into https://github.com/ckan/ckan/pull/5541 as well...

**Comment by smotornyuk** [comment]
Yes, i think that merging pdf view into core is a great idea

File: ckanext/pdfview/theme/templates/package/snippets/resource_view.html [hunk 1]
```diff
@@ -30,34 +30,44 @@
...
+          {% set action = 'view' %}
+      {% else %}
+          {% set controller = 'package' %}
+          {% set action = 'resource_view' %}
+
+      {% endif %}
+
       {% if not to_preview %}
-        {% set current_filters = request.str_GET.get('filters') %}
+          {% set current_filters = (request.GET|default(request.args)).get('filters') %}
```
Comment (EricSoroos) [inline ckanext/pdfview/theme/templates/package/snippets/resource_view.html]: Should this be h.get_request_param?

Comment (smotornyuk) [reply ckanext/pdfview/theme/templates/package/snippets/resource_view.html]: Good point

**Comment by pdelboca** [comment]
@smotornyuk I'm testing this locally and seems to work okay, just needs a small fix on the templates because the CKAN version comparison breaks with master.

Also, we can migrate from travis to Github Actions as we did in ckanext-harvester and [ckanext-showcase](https://github.com/ckan/ckanext-showcase/pull/95/commits/fddd37546dfb05761ded33ff08dc1d4212e53723) since it is cleaner and we don't need to workarounds to fix the postgresql versions. 

If you give me access to push commits to the branch I can wrap up this PR.

**Comment by amercader** [comment]
Closed in favour of https://github.com/ckan/ckanext-pdfview/pull/40

**Pull request closed without merging**